ui/app/components/clients/token/monthly-new.hbs

```diff
@@ -3,29 +3,31 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="chart-wrapper stacked-charts" data-test-chart="monthly new">
```

Comment:
Display is unchanged, just leverages component to consolidate styling
<img width="1060" alt="Screenshot 2024-01-23 at 12 23 07 PM" src="https://github.com/hashicorp/vault/assets/68122737/b82cf22b-7a8d-43a1-b7b8-e4684cf9b4c7">

Reply:
<img width="1054" alt="Screenshot 2024-01-23 at 1 11 19 PM" src="https://github.com/hashicorp/vault/assets/68122737/e619a8f4-51e9-440f-a710-f73c943e9a32">